src/test/scala/no/ndla/searchapi/TestData.scala

```diff
@@ -892,41 +892,55 @@ object TestData {
   )
 
   val resources = List(
-    Resource("urn:resource:1",
-             article1.title.head.title,
-             Some(s"urn:article:${article1.id.get}"),
-             Some("/subject:1/topic:1/resource:1"),
-             visibleMetadata),
-    Resource("urn:resource:2",
-             article2.title.head.title,
-             Some(s"urn:article:${article2.id.get}"),
-             Some("/subject:1/topic:1/resource:2"),
-             visibleMetadata),
-    Resource("urn:resource:3",
-             article3.title.head.title,
-             Some(s"urn:article:${article3.id.get}"),
-             Some("/subject:1/topic:3/resource:3"),
-             visibleMetadata),
-    Resource("urn:resource:4",
-             article4.title.head.title,
-             Some(s"urn:article:${article4.id.get}"),
-             Some("/subject:1/topic:1/topic:2/resource:4"),
-             visibleMetadata),
-    Resource("urn:resource:5",
-             article5.title.head.title,
-             Some(s"urn:article:${article5.id.get}"),
-             Some("/subject:2/topic:4/resource:5"),
-             visibleMetadata),
-    Resource("urn:resource:6",
-             article6.title.head.title,
-             Some(s"urn:article:${article6.id.get}"),
-             Some("/subject:2/topic:4/resource:6"),
-             visibleMetadata),
-    Resource("urn:resource:7",
-             article7.title.head.title,
-             Some(s"urn:article:${article7.id.get}"),
-             Some("/subject:2/topic:4/resource:7"),
-             visibleMetadata),
+    Resource(
+      "urn:resource:1",
+      article1.title.head.title,
+      Some(s"urn:article:${article1.id.get}"),
+      Some("/subject:3/topic:5/resource:1"),
```

Comment:
En gyldig path for ressursen, men en path som tidligere førte til at ressursen blei indeksert uten kontekster.